feed.html

```diff
@@ -0,0 +1,244 @@
+<!DOCTYPE html>
```

Comment:
According to the brief, the route to the feed page should be `/feed/index.html`. You need to have a feed folder and have an index.html file in it. The same thing is expected for the profile page. See this https://content.noroff.dev/css-frameworks/ca.html#brief